### PR TITLE
Add Swedish sv-SE (already in simple-spellchecker)

### PR DESCRIPTION
**Pull request by petermcj**
Before submitting, please confirm you've
 - [x] read and understood our [Contributing Guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md)
 - [x] completed [Mattermost Contributor Agreement](http://www.mattermost.org/mattermost-contributor-agreement/)
 - [x] executed `npm run lint:js` for proper code formatting

Please provide the following information:

**Summary**
Add swedish sv-SE already in simple-spellchecker

### CODE REVIEW

File: test/specs/spellchecker_test.js [hunk 1]
```diff
@@ -37,6 +37,9 @@ describe('main/Spellchecker.js', function() {
       SpellChecker.getSpellCheckerLocale('ru').should.equal('ru-RU');
       SpellChecker.getSpellCheckerLocale('ru-RU').should.equal('ru-RU');
 
+      SpellChecker.getSpellCheckerLocale('sv').should.equal('sv-SE');
+      SpellChecker.getSpellCheckerLocale('sv-SE').should.equal('sv-SE');
+
```
Comment (Willyfrog) [inline test/specs/spellchecker_test.js:42]: can you also add a sample of spellchecking corrections that should work with this language? see line 45 and below for examples of it.

**Review by Willyfrog — CHANGES_REQUESTED** [review]
hey! thanks for contributing a new spellchecker language. while I'm good accepting this PR (once you add  the change requested for tests) I'm not entirely sure it will make it for a release as we are going to change the spellchecker in the next version (if all goes as planned as it is a big change), but if we do a intermediate release it will definitely appear there.

let me know if you still want to contribute this, and if so can you add the requested test?d

File: test/specs/spellchecker_test.js (hunk 1 above)
Comment (petermcj) [reply test/specs/spellchecker_test.js:42]: Thanks for the info about the change in spellchecker and I hope that Swedish spellchecking also makes into your new “big change”. We are using Mattermost at work and we got annoyed using the desktop version with Swedish text (works fine in browser).  Saw that 5.32 server got Swedish as supported language great news! I have added test for sv-SE but circleci test seems to been stopped for some other reason than my changes.

Comment (Willyfrog) [reply test/specs/spellchecker_test.js:42]: yep, sometimes a couple of test might fail. I'm rerunnintg them now to see if that passes.

**Review by Willyfrog — APPROVED** [review]
LGTM! thanks!

**Review by devinbinnie — APPROVED** [review]
LGTM! Thanks @petermcj!

**Pull request merged**